pages/api/search.ts

```diff
@@ -0,0 +1,35 @@
+import db from '../../server/database/db';
+
+const Search = async (request, response) => {
+  const { ascendancy } = request.query;
+
+  db('build')
```

Comment:
@sanghin Apparently it is possible to add type here by doing:

```
db<Build>('build')
```
see http://knexjs.org/#Builder-knex


But it's not working when running `yarn build`.

Reply:
That's probably because we have to setup typescript ["server side" manually"](https://github.com/zeit/next.js/tree/canary/examples/custom-server-typescript)